onebusaway-android/src/main/java/org/onebusaway/android/map/MapModeController.java

```diff
@@ -66,12 +69,20 @@
 
         void showStops(List<ObaStop> stops, ObaReferences refs);
 
+        void showBikeStations(List<BikeRentalStation> bikeStations);
+
+        void clearBikeStations();
+
         boolean setMyLocation(boolean useDefaultZoom, boolean animateToLocation);
 
         void notifyOutOfRange();
 
         // Zooms to the region bounds, if a region has been set
         void zoomToRegion();
+
+        LatLng getSouthWest();
```

Comment:
@carvalhorr We can't actually include the `LatLng` object in MapModeController, because it's proprietary to Google, and we're supporting Amazon as well.  Anything that has a `com.google.android.gms.*` import has to be implemented in the `onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2` package classes.  In this controller interface, you could change it to `Location getSouthWest();`, because Location is part of the Android framework (AOSP) and not proprietary to Google Maps API v2.

Every time you push a new commit to the PR Travis CI should build the project, and it will fail when building the Amazon flavor for this type of problem - see https://travis-ci.org/OneBusAway/onebusaway-android/builds/246223817 for an example.  The Github PR will have a red X for any build failures.  Travis builds were temporarily all failing due to https://github.com/OneBusAway/onebusaway-android/issues/779, but I believe that's fixed now (still confirming).  So if you fix this, I believe the PR will give you a green dot for a build success.